src/physics.cpp

```diff
@@ -1160,4 +1166,65 @@ void sample_secondary_photons(Particle& p, int i_nuclide)
   }
 }
 
+// split a particle on the basis of the statistical weight
+// of the current location	
+void split_particle(Particle& p)
+{
+  // skip dead or no energy
+  if (p.E() <= 0 || !p.alive()) return;
+	
+  // Particle's position, weight and energy
+  Position pos  = p.r();
+  double weight = p.wgt();
+  double E = p.E();	      
+
+  // Check if this particle is in the weight weindow mesh and get the mesh bins in each direction
+  int ijk[3] = {0};     // mesh bin in each direction
+  bool in_mesh;  
+  // this is likely expensive 
+  // would like a more generic wrapper in case of cell based
+  // ww's
+  settings::ww_settings->mesh_->get_indices(pos, ijk, &in_mesh);
+  // no vr if not inside the mesh
+  if (!in_mesh) {
+    std::cout << "particle not in mesh" << std::endl;
```

Comment:
This should probably have a more descriptive message, and should instead use an `openmc::warning` instead. Would be good to say we're talking about the VR mesh.